snap/hooks/install

```diff
@@ -1,28 +1,42 @@
 #!/bin/bash
+# Installation hook for charmed-mongodb snap
 
 set -eux
 
+export CONF="${SNAP_DATA}/etc/mongod"
+export PBM_CONF="${SNAP_DATA}/etc/pbm/"
+export DATA="${SNAP_COMMON}/var/lib/mongodb"
+export LOG="${SNAP_COMMON}/var/log/mongodb"
+
+# If the $DATA dir is not there yet, create it
+if [ stat -c '%u' "$DATA" == 0 -o ! -d "$DATA" ]; then
+    mkdir -p $DATA
+    mkdir -p $LOG
+    chmod g+s "${LOG}"/*
+    chown -R 584788:root "${SNAP_COMMON}"/*
+    chgrp root "${LOG}"*
+else
+    mkdir -p $LOG
+    chmod g+s "$LOG"*
+    chgrp root "${LOG}"*
+fi
```

Comment:
A few questions
1. does it hurt to do chown/chgrp/chmod more than once, if not perhaps they can be moved out of this if/else
2. maybe this logic can be simplified, it seems that no matter what

```
    mkdir -p $LOG
    chmod g+s "$LOG"*
    chgrp root "${LOG}"*
 ```

gets called since it is in both if statements?

Reply:
I guess it can be done.
I'm currently testing manually so I'll also ensure that we are not breaking things this way :)

Reply:
I'm still checking but I might need some more changes so I'll go on checking everything